### PR TITLE
Add more test cases for multiple WDT model file ordering

### PR DESCRIPTION
**Pull request by doxiao**
Add two more test cases for multiple WDT model file ordering. One is to have two model files in the image and another is to have multiple model files in both image and configmap.
Also refactor existing ItMiiConfigMap and ItMiiConfigMapOverride to use the new common test util method for creating a config map.

External Jenkins job is clean in kind cluster: https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/207/.

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java [hunk 1]
```diff
@@ -29,63 +26,94 @@
...
 import static oracle.weblogic.kubernetes.actions.ActionConstants.MODEL_DIR;
-import static oracle.weblogic.kubernetes.actions.TestActions.createConfigMap;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteImage;
 import static oracle.weblogic.kubernetes.actions.TestActions.getServiceNodePort;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkPodReady;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkServiceExists;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createConfigMapAndVerify;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDockerRegistrySecret;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDomainAndVerify;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createMiiImageAndVerify;
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:54]: imported twice

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java [hunk 2]
```diff
@@ -193,23 +404,14 @@ public void testCreateMiiDomainWithMultiModeCM() {
...
-            domainUid, domainNamespace, dsName3, maxCapacityValue, "5"));
+  }
 
-    logger.info(String.format("Domain %s in namespace %s DataSource %s MaxCapacity is %s, as expected",
-            domainUid, domainNamespace, dsName3, "5"));
+  @AfterAll
+  public void tearDownAll() {
+    // delete the domain image created in the test class
+    if (miiImageMultiModel != null) {
+      deleteImage(miiImageMultiModel);
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:413]: will this work if the domains are running?

Comment (doxiao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:413]: Probably not. Let me check. If not, I'll delete the domain first.

Comment (doxiao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: Added delete domains before delete the image.

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: I see deleting image works before deleting the domain..

Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: not sure if domain deletion will have issues though

Comment (doxiao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: With the change, we are doing what we do in ItMiiDomain.

Comment (doxiao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: In addition, I tested the current version and found no problem and the image was removed.

Comment (doxiao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: I re-tested both cases, and either way worked without problem:).

Comment (doxiao) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiMultiModel.java:431]: I recall that a Docker image cannot be deleted if there are containers using it. The containers have to be stopped first. So I think it is probably safer to delete the domain before deleting the image.

**Comment by doxiao** [comment]
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/203/ was before the last submit.

**Comment by doxiao** [comment]
The latest version are clean on Jenkins too:
https://build.weblogick8s.org:8443/view/all/job/weblogic-kubernetes-operator-kind-new/228
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests/797

**Pull request merged**